asyncapi/json-schemas/pe/sales/sla/sales-sla.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://schemas.ruter.no/adt/ota/api/v3.0/pe/sales/diagnostics/sales-diagnostics.json",
+  "type": "object",
+  "title": "SalesSla",
+  "required": [
+    "eventTimestamp",
+    "traceId",
+    "spanId",
+    "metrics",
+    "context"
+  ],
+  "additionalProperties": true,
+  "properties": {
+    "eventTimestamp": {
+      "$id": "#/properties/eventTimestamp",
+      "type": "string",
+      "description": "Time of diagnostics generated",
+      "format": "date-time"
+    },
+    "traceId": {
+      "$id": "#/properties/traceId",
+      "type": "string",
+      "description": "A unique identifier for the transaction. Also used to detect duplicate messages received. Matches the traceId of the incoming current_stop message"
+    },
+    "spanId": {
+      "$id": "#/properties/spanId",
+      "type": "string",
+      "description": "A unique identifier for an individual step of a transaction. Consists of the spanId from the current_stop message and a new UUID for this step, separated by a colon."
+    },
+    "metrics": {
+      "$id": "#/properties/metrics",
+      "type": "object",
+      "description": "Contains measurable diagnostics fields",
+      "required": [
+        "loggedIn"
+      ],
+      "additionalProperties": true,
+      "properties": {
+        "loggedIn": {
+          "$id": "#/properties/metrics/properties/loggedIn",
+          "type": "boolean",
+          "description": "Whether or not the user is logged in to RuterSalg"
+        }
+      }
+    },
+    "context": {
+      "$id": "#/properties/context",
+      "type": "object",
+      "description": "Contains fields with context data",
+      "required": [
+        "vehicleRef",
+        "progressRef",
+        "lineRef",
+        "assignmentRef",
+        "chainId",
+        "stopPlaceId",
+        "appVersion",
+        "userId"
+      ],
+      "additionalProperties": true,
+      "properties": {
+        "vehicleRef": {
+          "$id": "#/properties/context/properties/vehicleRef",
+          "type": "string",
+          "description": "A reference to the Progress event that triggered this message. Matches the traceId from the Progress event."
+        },
+        "progressRef": {
+          "$id": "#/properties/context/properties/progressRef",
+          "type": "string",
+          "description": "A reference to the Progress event that triggered this message. Matches the traceId from the Progress event."
+        },
+        "lineRef": {
+          "$id": "#/properties/context/properties/lineRef",
+          "type": "string",
+          "description": "A reference to the Progress event that triggered this message. Matches the traceId from the Progress event."
+        },
+        "assignmentRef": {
```

Comment:
hva er `assignmentRef`? Det hadde vi ikke i kladden

Reply:
Det er mer meta-data om kjøretøyets tur. Denne, som nesten alle de andre feltene, skal ligge i current_stop-meldingen og sendes videre derfra. Dette er for å gjøre det veldig eksplisitt for operatør hvor bruddet har foregått. Da slipper de (og Ruter/Tet) å slå opp slik info for et gitt kjøretøy på et gitt tidspunkt.